### ISSUE TITLE
feature request: range slider compatibility for Scattergl

### ISSUE THREAD

**Issue by lamourj** [plotly/plotly.js#2627]
As reported in #2609, plotting a Scattergl with a range slider is not supported yet. That would be a great feature to have 😄 

Thanks for the good work 👍

**Comment by Abhilash-Chandran** [comment]
Hi, Is anyone working on this issue..? I am not sure if I can help. But if any pointer on where to look for this issue is given, I will give a try to find a fix for this issue. I kind of depend on 'scattergl' plot with a range slider in my project. So I should at least try to find a fix for this issue.

**Comment by smitabisoyi** [comment]
Any update on this feature?
Thanks for the good work.

**Comment by travlytle** [comment]
Just wanted to check and see if this feature ever has gotten implemented?

I currently don't see it working.
![image](https://user-images.githubusercontent.com/1254680/84409065-ec115300-abda-11ea-9617-0d61e69a9606.png)

pip freeze | findstr plotly
plotly==4.8.1

pip freeze | findstr dash
dash==1.12.0
dash-core-components==1.10.0
dash-html-components==1.0.3
dash-renderer==1.4.1
dash-table==4.7.0
jupyter-dash==0.2.1.post1

**Comment by Williangalvani** [comment]
I just spent a couple of hours digging around the code fruitlessly... 

I can tell that plot() returns early here: https://github.com/plotly/plotly.js/blob/master/src/traces/scattergl/plot.js#L47 but that doesn't mean a lot to me. I tried hacking a sceneUpdate() in there but it didn't change anything.

Anyone got any tips/directions?

**Comment by FelixWaigner** [comment]
Any updates for the possibility to enable range slider for scattergl?

**Comment by sleighsoft** [comment]
+1 for this feature

**Comment by DanRos22** [comment]
+1 for this feature

**Comment by cdalvaro** [comment]
It would be great to have this feature working!

**Comment by nicolaskruchten** [comment]
Agreed! No one is working on this item at the moment but we would certainly welcome a community contribution for it!

**Comment by wesley-cheek** [comment]
Bump for somebody to implement this feature!

**Comment by rsamer** [comment]
+1 for this feature

**Comment by wmachadoa** [comment]
Are there any updates/workarounds concerning this? It's very necessary in my case!

**Comment by jarrisoncano** [comment]
+1 for this feature

**Comment by markchagers** [comment]
+1

**Comment by karimtamerBW** [comment]
Any updates please, really need this?

**Comment by msteckle** [comment]
Looking forward to this!!!

**Comment by ntaylor-ads** [comment]
+1

**Comment by ypprado** [comment]
+1

**Comment by TheSisb** [comment]
+1

**Comment by Youenlb** [comment]
+1
